testing/html/testing_page.html

```diff
@@ -301,10 +311,26 @@ <h3 id="switchabletesting-2">bartest</h3>
         <td><input name="widget3" type="text" value="widget3"></td>
       </tr>
       <tr hidden>
-        <td colspan="5"><p>some hidden content row 3</p></td>
+        <td colspan="5">
+          <p>some hidden content row 3</p>
+        </td>
       </tr>
     </tbody>
   </table>
   <iframe src="iframe_page.html" height="600" width="800" name="some_iframe"></iframe>
+  <br />
+  <div id="ouia">
```

Comment:
Can you add a `data-ouia-safe` attribute to two of these elements, one `true` and one `false`, so that we can have unit test coverage of the `is_safe` property behavior?

Reply:
Done